onnx2kerastl/reshape_layers.py

```diff
@@ -424,9 +427,11 @@ def convert_expand(node, params, layers, lambda_func, node_name, keras_name):
 
     input_0 = ensure_tf_type(layers[node.input[0]], name="%s_const" % keras_name)
     input_1 = ensure_numpy_type(layers[node.input[1]]).astype(np.int32)
-
+    if input_0.dtype.is_bool:
+        input_0 = tf.cast(input_0, dtype=input_1.dtype)
```

Comment:
should the type here be just `int` instead of `input_1.dtype`? i think maybe in some cases `input_1.dtype` can be float or other non int and we would still want `input_0` to be casted to `int`